CHANGELOG.md

```diff
@@ -4,10 +4,16 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.0.1]
+### Created
+- `SECURITY.md` file creation.
+- GitHub Actions Workflow Creation. 
+- Updated dependenices resolving security problems. 
```

Comment:
I have put this here as it's for Jim's PR 👍

Reply:
Thanks Nick.